server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -109,7 +106,6 @@ public Entry(Snapshot snapshot, boolean includeGlobalState, boolean partial, Sta
             } else {
                 this.shards = shards;
                 this.waitingIndices = findWaitingIndices(shards);
-                assert assertShardsConsistent(state, indices, shards);
```

Comment:
This method is still present in latest ES master, why did you remove it?

Reply:
Oh, my bad. It was not present at the time of original pr https://github.com/original-brownbear/elasticsearch/blob/808b8133cf9a7baf58615f0f00ddab402e448169/server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java It must have been added later. I didn't compare to master. I will add it back.